docs/release-notes/next/PT-165993503-fix_bug_in_min_gas_price.md

```diff
@@ -0,0 +1,2 @@
+* Fixes the mempool minimum fee (configured by miner) entrancy check for contract transactions, they incorrectly included the
```

Comment:
I see this follows https://github.com/aeternity/aeternity/pull/2424/files#r283872416